app/views/draws/show.html.erb

```diff
@@ -19,7 +19,10 @@
   <% end %>
 </div>
 <% if policy(@draw).activate? %>
-  <%= link_to 'Begin draw process', activate_draw_path(@draw), method: :patch %>
+  <%= link_to 'Begin draw process<br>'.html_safe, activate_draw_path(@draw), method: :patch %>
+<% end %>
+<% if policy(@draw).update? %>
```

Comment:
We're going to want to replace this with `policy(@draw).group_actions?` after #165 is merged in.